maestro-client/src/main/java/maestro/Maestro.kt

```diff
@@ -545,10 +545,10 @@ class Maestro(private val driver: Driver) : AutoCloseable {
         }
     }
 
-    fun setLocation(latitude: Double, longitude: Double) {
+    fun setLocation(latitude: String, longitude: String) {
```

Comment:
Is this necessary? Conceptually, it's a double, even if we got it from env var. But maybe it's justified.

(I'm new to the codebase)

Reply:
@bartekpacia This is needed to as the command class data type is changed to string. Otherwise compilation error.